eZ/Bundle/EzPublishCoreBundle/Features/Context/ContentPreviewContext.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\Features\Context;
+
+use Behat\Behat\Context\Context;
+use Behat\Behat\Context\SnippetAcceptingContext;
+use eZ\Publish\API\Repository\Values\Content\Content;
+use eZ\Publish\API\Repository\Values\Content\VersionInfo;
+use EzSystems\BehatBundle\Context\Browser\Context as BrowserContext;
+use PHPUnit_Framework_Assert as Assertion;
+
+class ContentPreviewContext extends BrowserContext implements Context, SnippetAcceptingContext
+{
+    /**
+     * Items referenced by key sentences.
+     * @var array
+     */
+    private $_;
```

Comment:
This is used to store phrase references. Yes, it can be named differently, it just felt right in that particular context.

I'll write it the way you instructed.